database/suitapp_db.sql

```diff
@@ -62,6 +62,17 @@ CREATE TABLE follows
     time_created TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP 
 );
 
+CREATE TABLE events
+(
+    id SERIAL PRIMARY KEY,
+    start_time VARCHAR NOT NULL,
```

Comment:
for `start_time` and `end_time` use `TIMESTAMP WITH TIME ZONE` instead of `VARCHAR`
and when you seed the table insert values like ` '2004-10-19 10:23:54+02' `